allocator/nonmoving_test.go

```diff
@@ -16,7 +16,7 @@ func TestNonMoving(t *testing.T) {
 		{
 			name: "native",
 			mem:  NewNonMoving().Allocate(10, 20),
-			cap:  int(pageSize),
+			cap:  int(pageSize), //nolint:unconvert // depending on platform, not an int
```

Comment:
Is this needed? It can probably be an untyped constant on Windows, and an `int` elsewhere.

Reply:
Unless there's some technique I don't know, a `const foo = 12414` is always an `int` (a literal is implicitly typed by the expression). So I went with using `int` on windows like elsewhere and convert to `uint64` in the logic which seems fine

Reply:
Constants in Go are actually untyped, until you coerce them. Untyped integer constants just default to `int`, but they can be used in any integer context without conversion.